frontend/src/metabase/home/components/HomeRecentSection/HomeRecentSection.unit.spec.tsx

```diff
@@ -33,22 +34,12 @@ const setup = async ({ user = createMockUser() }: SetupOpts = {}) => {
 };
 
 describe("HomeRecentSection", () => {
-  beforeEach(() => {
-    jest.useFakeTimers({ advanceTimers: true });
-    jest.setSystemTime(new Date(2020, 0, 4));
-  });
-
-  afterEach(() => {
-    jest.useRealTimers();
-    jest.restoreAllMocks();
-  });
-
   describe("new installers", () => {
     it("should show a help link for new installers", async () => {
       await setup({
         user: createMockUser({
           is_installer: true,
-          first_login: "2020-01-05T00:00:00Z",
+          first_login: dayjs().add(1, "hour").toISOString(),
```

Comment:
Fake timers don't seem to work well with RTK 2, see https://github.com/reduxjs/redux-toolkit/issues/1762.
It's not possible to use legacy timers because the original implementation of this test uses `jest.setSystemTime`